src/deepsparse/loggers/async_logger.py

```diff
@@ -43,19 +43,28 @@ def __init__(self, logger: BaseLogger, max_workers: int = 1):
         self.logger = logger
         self._job_pool: Executor = ThreadPoolExecutor(max_workers=max_workers)
 
-    def log(self, identifier: str, value: Any, category: MetricCategories):
+    def log(
+        self,
+        identifier: str,
+        value: Any,
+        category: MetricCategories,
+        group: Optional[SystemMetricGroups] = None,
```

Comment:
why do we need an explicit group? this also hardcodes an optional variable specifically for system logs which seems less than ideal. any reason to not log as a `system` log, with the identifier as something like `system/{GROUP}/name`?